core/text/formatters.py

```diff
@@ -421,6 +375,18 @@ def formatter_immune(m) -> ImmuneString:
     return ImmuneString(str(value))
 
 
+def get_all_phrase_and_prose_formatters() -> Tuple[Dict[str, str], Dict[str, str]]:
```

Comment:
Per the discussion in the meetup, does this work?

```suggestion
def get_all_phrase_and_prose_formatters() -> tuple[dict[str, str], dict[str, str]]:
```

If so, let's use that everywhere because it doesn't require the type import, so it's easier to write. I can try see if we can make a new rule for it

Reply:
Seems to work.